setup.py

```diff
@@ -18,6 +18,9 @@
         'python-magic;platform_system!="Windows"',
         'python-magic-bin;platform_system=="Windows"',
     ],
+    extras_require={
+        "bootstrap": ["bootstrap>=5.0"],
```

Comment:
I think you want ims-bootstrap>=5.0

Reply:
I think you'll want to exclude v6 as well so it doesn't upgrade major versions:

```
"bootstrap": ["bootstrap>=5.0,<6.0"],
```

Reply:
This is unresolved